frontend/plan/src/components/selector/CourseDetails.js

```diff
@@ -43,6 +44,16 @@ export default function CourseDetails({ course, getCourse }) {
                     </li>
                 ) : null
             }
+            {prerequisites.length ? (
+                <li>
+                    <span className="icon is-small">
+                        <i className="fas fa-random" />
+                    </span>
+                    &nbsp; Prerequisites: &nbsp;
+                    <TagList elements={[prerequisites.replace(/-/g, " ")]} limit={2} />
```

Comment:
Will there ever be more than one tag?

Reply:
I've actually written some code to extract course codes out of the prereq text because there were cases where the prereq text was really, really long and overflowed out of the tag. I think for prereqs, we should always show a show more button that shows the original text if the text more than ~1.5 times the length of the total course code lengths, though I'd like design to comment on that first.